bin/ch-test

```diff
@@ -111,12 +111,20 @@ builder_set () {
 dir_img_mk () {
     dir_img_rm
     printf "creating %s\n\n" "$CH_TEST_IMGDIR"
-    mkdir "$CH_TEST_IMGDIR"
+    if [[ -n $SLURM_JOBID ]]; then
+        srun --ntasks-per-node=1 mkdir "$CH_TEST_IMGDIR"
+    else
+        mkdir "$CH_TEST_IMGDIR"
+    fi
 }
 
 dir_img_rm () {
     echo "removing $CH_TEST_IMGDIR"
-    rm -rf --one-file-system "$CH_TEST_IMGDIR"
+    if [[ -n $SLURM_JOBID ]]; then
+        srun --ntasks-per-node=1 rm -rf --one-file-system "$CH_TEST_IMGDIR"
+    else
+        rm -rf --one-file-system "$CH_TEST_IMGDIR"
+    fi
```

Comment:
Especially now that it's going through `srun`, good I think to validate that `$CH_TEST_IMGDIR` and `$CH_TEST_TARDIR` are what we think they are, or empty, before `rm -rf`ing them.

Reply:
Any suggestions how to do this effectively without creating situations where the check produce false negatives? I would like to avoid situations where `ch-test clean` doesn't do anything because the check fails due to missing artifacts caused by forced test-suite cancel.

Reply:
Maybe a sentinel file? Or, if the directory is empty, we also remove it, since that's low cost.